gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -169,12 +171,25 @@ def _merge_headers(known_args, pipeline_args, pipeline_mode):
     else:
       headers |= vcf_header_io.ReadVcfHeaders(known_args.input_pattern)
 
-    _ = (headers
-         | 'MergeHeaders' >> merge_headers.MergeHeaders(
-             known_args.split_alternate_allele_info_fields)
-         | 'WriteHeaders' >> vcf_header_io.WriteVcfHeaders(
-             known_args.representative_header_file))
-
+    merged_headers = (headers
+                      | 'MergeHeaders' >> merge_headers.MergeHeaders(
+                          known_args.split_alternate_allele_info_fields))
+    if known_args.robust_header_extraction:
+      headers_from_variants = (
```

Comment:
Consider extracting the body of this `if` as separate function that just returns a merged_header.

Reply:
Done